[2024-11-09T14:23:43.323+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-09T14:23:43.348+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline.fetch_data_from_json manual__2024-11-09T14:23:35.642340+00:00 [queued]>
[2024-11-09T14:23:43.355+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline.fetch_data_from_json manual__2024-11-09T14:23:35.642340+00:00 [queued]>
[2024-11-09T14:23:43.356+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-11-09T14:23:43.368+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): fetch_data_from_json> on 2024-11-09 14:23:35.642340+00:00
[2024-11-09T14:23:43.371+0000] {standard_task_runner.py:64} INFO - Started process 215 to run task
[2024-11-09T14:23:43.374+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipeline', 'fetch_data_from_json', 'manual__2024-11-09T14:23:35.642340+00:00', '--job-id', '328', '--raw', '--subdir', 'DAGS_FOLDER/pipline.py', '--cfg-path', '/tmp/tmp752oo_eu']
[2024-11-09T14:23:43.377+0000] {standard_task_runner.py:91} INFO - Job 328: Subtask fetch_data_from_json
[2024-11-09T14:23:43.421+0000] {task_command.py:426} INFO - Running <TaskInstance: pipeline.fetch_data_from_json manual__2024-11-09T14:23:35.642340+00:00 [running]> on host 66156c2b6481
[2024-11-09T14:23:43.504+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pipeline' AIRFLOW_CTX_TASK_ID='fetch_data_from_json' AIRFLOW_CTX_EXECUTION_DATE='2024-11-09T14:23:35.642340+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-09T14:23:35.642340+00:00'
[2024-11-09T14:23:43.505+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-09T14:23:43.507+0000] {logging_mixin.py:188} INFO - Error reading JSON file: [Errno 2] No such file or directory: 'C:/Users/mohammed/Desktop/Airflow/Data-Engineering-Pipeline/***/data_set/journals.json'
[2024-11-09T14:23:43.507+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-11-09T14:23:43.508+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-09T14:23:43.516+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipeline, task_id=fetch_data_from_json, run_id=manual__2024-11-09T14:23:35.642340+00:00, execution_date=20241109T142335, start_date=20241109T142343, end_date=20241109T142343
[2024-11-09T14:23:43.548+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-11-09T14:23:43.565+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-09T14:23:43.568+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
